Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Basic docker based environment
+# Necessary to trick dokku into building the documentation
+# using dockerfile instead of herokuish
+FROM ubuntu:17.04
+
+# Add basic tools
+RUN apt-get update && \
+  apt-get install -y build-essential \
+    software-properties-common \
+    curl \
+    git \
+    libxml2 \
+    libffi-dev \
+    libssl-dev
+
+RUN LC_ALL=C.UTF-8 add-apt-repository ppa:ondrej/php && \
+  apt-get update && \
```

Comment:
these 2 lines can be moved to the RUN above, and in fact, this `RUN` could be merged with the one above. This would make the image smaller and probably faster to build